src/Plugin/Field/FieldWidget/ReactParagraphs.php

```diff
@@ -65,7 +67,7 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function settingsSummary() {
+  public function settingsSummary(): array {
```

Comment:
Oh, are we going to start this now as a standard? We should update our linting files to catch these if they are missing.

Reply:
My phpstorm wass suggesting it and I figured it was good practice.